doc/guides/backporting-to-release-lines.md

```diff
@@ -74,6 +74,8 @@ hint: and commit the result with 'git commit'
    4. In the description add a reference to the original PR
    5. Run a [`node-test-pull-request`][] CI job (with `REBASE_UNTO` set to the
       default `<pr base barnch>`)
+10. If during the review process conflicts arise, use the following to rebase:  
+    `git pull --rebase upstream v6.x-staging`
```

Comment:
Maybe we better use `v6.x` in all the examples instead of `v7.x`? The latter is in the maintenance phase, there will be hardly any backports for it.